quickstarts/blueprint-camel-drools-decision-table/pom.xml

```diff
@@ -65,9 +65,9 @@
             <url>https://repository.jboss.org/nexus/content/groups/ea</url>
         </repository>
         <repository>
-          <id>redhat-product-ga</id>
-          <name>Red Hat Product GA</name>
-          <url>https://maven.repository.redhat.com/nexus/content/groups/product-ga</url>
+          <id>redhat-product-bxms-releases</id>
+          <name>Red Hat BXMS Product Releases</name>
+          <url>http://download.eng.bos.redhat.com/brewroot/repos/jb-bxms-6.2-build/latest/maven/</url>
```

Comment:
We really should not depend on this repository. It's internal one and not visible outside of the VPN. If needed we should add this to settings.xml to test release candiates. The final artifacts will then be available in the maven.repository.redhat.com.